app/views/reports/regions/show.html.erb

```diff
@@ -364,7 +364,7 @@
                   <p class="m-0px c-black c-print-black">
                     total <%= "registration".pluralize(@data.last_value(:cumulative_registrations)) %><br>till
                     <span data-registrations-period-end="<%= @period.bp_control_range_end_date %>">
-                      <%= @period.bp_control_range_end_date %>
+                      <%= Date.today.to_s(:day_mon_year) %>
```

Comment:
This won't be correct if they have chosen a date in the past from the dropdown.

Reply:
That's a good point @rsanheim. should I check if the period is the current period? If it isn't I set the `end_date`, if it is, I set the current date?

Reply:
I decided to use a conditional to check the period selected to show either the current date or the end of the month of the selected date